potpie/context-engine/src/potpie_context_engine/adapters/outbound/resources/local_chunk_store.py

```
@@ -0,0 +1,547 @@
+"""Local filesystem chunk store under ~/.potpie/resources/."""
+
+from __future__ import annotations
+
+import hashlib
+import json
+import shutil
+import uuid
+from dataclasses import dataclass, field
+from pathlib import Path
+from typing import Any, Iterable
```

Ruff (0.16.2)

[warning] Import from `collections.abc` instead: `Iterable`

Import from `collections.abc`

(UP035)